features/aanschrijfwijze/dev/geslachtswijziging.feature

```diff
@@ -0,0 +1,128 @@
+      | fields              | adressering.aanschrijfwijze.naam |
+      Dan heeft de response een persoon met alleen de volgende 'adressering' gegevens
+      | naam                          | waarde                    |
+      | aanschrijfwijze.naam          | <naam in aanschrijfwijze> |
+      | aanschrijfwijze.aanspreekvorm | <aanspreekvorm>           |
+
+      Voorbeelden:
+      | titel predicaat | geslacht | naam in aanschrijfwijze    | aanspreekvorm            |
+      | JH              | M        | Jonkheer A. van den Aedel  | De hoogwelgeboren heer   |
+      | JH              | V        | Jonkvrouw A. van den Aedel | De hoogwelgeboren vrouwe |
+      | JV              | M        | Jonkheer A. van den Aedel  | De hoogwelgeboren heer   |
+      | JV              | V        | Jonkvrouw A. van den Aedel | De hoogwelgeboren vrouwe |
+      | R               | M        | A. ridder van den Aedel    | De hoogwelgeboren heer   |
+      | G               | V        | A. gravin van den Aedel    | De hooggeboren vrouwe    |
+      | GI              | M        | A. graaf van den Aedel     | De hooggeboren heer      |
+      | P               | V        | A. prinses van den Aedel   | De hoogheid              |
+      | PS              | M        | A. prins van den Aedel     | De hoogheid              |
+
+    Abstract Scenario: Adellijke titel of predicaat "<titel predicaat>" en geslacht "<geslacht>" voert geen titel en geen aanspreekvorm
+      Gegeven de persoon met burgerservicenummer '000000115' heeft de volgende gegevens
+      | naam                        | waarde     |
+      | geslachtsaanduiding (04.10) | <geslacht> |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                                 | waarde            |
+      | voornamen (02.10)                    | Anne              |
+      | adellijke titel of predicaat (02.20) | <titel predicaat> |
+      | voorvoegsel (02.30)                  | van den           |
+      | geslachtsnaam (02.40)                | Aedel             |
+      | aanduiding naamgebruik (61.10)       | E                 |
+      En de persoon heeft nooit een actueel of ontbonden huwelijk of partnerschap gehad
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                           |
+      | type                | RaadpleegMetBurgerservicenummer  |
+      | burgerservicenummer | 000000115                        |
+      | fields              | adressering.aanschrijfwijze.naam |
+      Dan heeft de response een persoon met alleen de volgende 'adressering' gegevens
+      | naam                 | waarde           |
+      | aanschrijfwijze.naam | A. van den Aedel |
+
+      Voorbeelden:
+      | titel predicaat | geslacht |
+      | JH              | O        |
+      | JV              | O        |
+      | R               | V        |
+      | R               | O        |
+      | B               | M        |
```

Comment:
Waarom hier niet? Deze combinatie staat op regel 15 met titel en aanspreekvorm

Reply:
is fout, moet een "O" (geslacht onbekend) zijn. aangepast